### PR TITLE
RegionalDR example application

### PR DESCRIPTION
**Pull request by r2dedios**
Included Doc and Manifests for deploying a test application for checking the RegionalDR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ by [ACM][acm]'s heartbeat mechanism, a failover will be triggered to all related
 
 ## Further Reading
 
-- [Deploy an example DR Application][example-app]
+- [Deploy an example DR Application](./rdr-example-app/README.md)
```
Comment (TomerFi) [inline README.md:125]: Revert this to `[example-app]` and scroll down to the end of the README; there, you will find the link that needs to be modified.

image

**Review by TomerFi — CHANGES_REQUESTED** [review]

**Comment by TomerFi** [comment]
@r2dedios We don't need both example apps. We should delete the old _busybox_ one while PRing the new _pacman_ one.

Also, the previous example app folder structure was:
- [doc/](https://github.com/r2dedios/regional-resiliency-pattern/blob/pattern-final-fixes/doc/) - All documentation-related, i.e. images and markdowns.
- [doc/ExampleDRApp.md](https://github.com/r2dedios/regional-resiliency-pattern/blob/pattern-final-fixes/doc/ExampleDRApp.md) - The example app documenation.
- [doc/example-app](https://github.com/r2dedios/regional-resiliency-pattern/tree/pattern-final-fixes/doc/example-app) - The example app manifests.

The new example app diverged from this structure and was placed in the project's root. Should we adapt to the previous structure to be consistent with the rest of the pattern documentation?

**Comment by TomerFi** [comment]
For the failing markdown lint, you can run this locally to work through the errors:

```shell
podman run --privileged \
  -e RUN_LOCAL=true \
  -e DEFAULT_BRANCH=main \
  -e VALIDATE_MARKDOWN=true  \
  -e VALIDATE_NATURAL_LANGUAGE=true  \
  -v $PWD:/tmp/lint \
  ghcr.io/super-linter/super-linter:latest
```

**Comment by TomerFi** [comment]
@r2dedios I just noticed you created this PR from the branch `pattern-final-fixes,` which was already merged here: https://github.com/RHEcosystemAppEng/regional-resiliency-pattern/pull/5. (That explains 107 commits...).
image

You should refresh your fork, create a new branch from the main one, and only include commits relevant to this PR.

**Comment by r2dedios** [comment]
PR wrong created

**Pull request closed without merging**